docs/USAGE-GUIDE.md

```diff
@@ -495,6 +495,20 @@ To use OCSP stapling, both server and client must call `s2n_config_set_status_re
 
 The OCSP stapling information will be automatically validated if the underlying libcrypto supports OCSP validation. `s2n_config_set_check_stapled_ocsp_response()` can be called with "0" to turn this off. Call `s2n_connection_get_ocsp_response()` to retrieve the received OCSP stapling information for manual verification.
 
+### CRL Validation
+
+Certificate Revocation Lists (CRLs) are lists of issued, unexpired certificates that have been revoked by the CA. CAs publish updated versions of these lists periodically. A peer wishing to verify a certificate can obtain a CRL from the CA, validate the CRL to make sure it was issued by the CA, and check to ensure the given certificate’s serial number is not contained in the list, and thus has not been revoked by the CA.
```

Comment:
I believe this check is needed because all CRLs are obtained from CAs via HTTP, so you can't really be sure the CRL was retrieved from the CA or not. Also, there are cases where you indirectly obtain the CRL. Regardless, this is one of many CRL validation steps performed, so it may make sense for this to just say "obtain a CRL from the CA, validate the CRL, and check..."